content/doc/book/blueocean/activity.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: section
+title: Activity Viewer
+wip: true
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= The Blue Ocean Activity Viewer
+
+The Blue Ocean Activity Viewer shows the all activity related to one
+repository Pipeline.
+
+image:blueocean/activity/overview.png[Overview of the Dashboard, role=center]
+
+
+== Activity
+
+
+
+== Branches
+
+
+image:blueocean/activity/branches.png["Aborted" Status Icon, role=center]
+
+
+== Pull Requests
+
+Blue Ocean displays Pull Requests separately from branches,
+but otherwise the Pull Requests list behaves similar to the Branches list.
+
+image:blueocean/activity/pull-requests.png["Aborted" Status Icon, role=center]
+
+Clicking on a Pipeline will bring up the <<pipeline-run-details#, Pipeline Run Details>>
+for the latest completed or in-progress Run of a Pipeline.
+"In Progress" runs can be aborted from this list.
+Pull Request Pipelines whose latest run has completed can be run again.
+Pull Request Pipelines do not display <<dashboard#pipeline-health, "Heath Icons">>
+and cannot be edited or marked as favorites.
+
+NOTE: By default, when a Pull Request is closed,
+Jenkins will delete the Pipeline and runs for that Pull Request.
```

Comment:
This is correct but could be more correct. Pull Requests are marked as Orphans and are periodically cleaned up, they're not immediately deleted however.

In the May OSS Demo Day video from CloudBees Jesse and Stephen talked a little bit about this